### ISSUE TITLE
Windows 11 disconnect after few seconds

### ISSUE THREAD

**Issue by FC2469** [immense/Remotely#402]
Unable to remote control Windows 11. Remote session lost after few seconds

**Comment by nexius76** [comment]
have you checked the exception in Windows Firewall?

**Comment by FC2469** [comment]
I try with windows firewall disabled! In my notebook with Windows 11 work fine, in workstation disconnect after few seconds. In event viewer I see error 0xc0000409 from C:\Program Files\Remotely\Desktop\mrwebrtc.DLL

**Comment by nexius76** [comment]
The error seems refer to a filesystem failure or wrong installation or wrong registry entry. First of all try to connect to your workstation disabling webrtc (that seems to be the problem). If it works, try to reinstall remotely agent.

**Comment by nexius76** [comment]
![immagine](https://user-images.githubusercontent.com/18268547/142994101-618df5a7-1547-4983-b003-1ce9e3fe9616.png)

**Comment by FC2469** [comment]
Yes, without webrtc work, but slower in some times

**Comment by nexius76** [comment]
But I suppose it's another issue. Try to reinstall the agent on the workstation.
Are you in the same network? I mean, the workstation is in the same network of the server? Can you describe your environment and the way you would like things sould work?

**Comment by FC2469** [comment]
https://github.com/microsoft/MixedReality-WebRTC/issues/588 seem issue is in DLL need updated 
For answering to your question, my server is not in same network, but remote. With my notebook work fine. Currently I try this solution (I have try unsuccessful other solutions because poor connection speed and performance)

**Comment by nexius76** [comment]
are your notebook and workstation in the same network?

**Comment by FC2469** [comment]
yes, but also I try with cellular phone with my notebook disconnected from my network. Same result. If with workstation I try to connect to my notebook, this work. Is not very important, because I try software. I think for best compatibility, need to set use webrtc setting available also for single machine in case of troubles.

**Comment by nexius76** [comment]
Ok, if you just need to disable webrtc only for the workstation, you can disable webrtc from the workstation tab in the server. So you can use webrtc by default and disable it for specific machine.
Did you try reinstall remotely agent?

**Comment by nexius76** [comment]
![immagine](https://user-images.githubusercontent.com/18268547/142998669-1bfb4e5b-09ee-4bdb-aaa7-86783ca5af50.png)

**Comment by FC2469** [comment]
Yes, uninstalled, reboot machine, reinstalled, but nothing!

**Comment by MazixM** [comment]
I had the same problem on one of my laptops. I'm not sure why, but when I build new exe from source code it start working fine.
After build exe was around 8MB. Maybe it can help you..

**Comment by FC2469** [comment]
yes, if compile and use compiled version, this work

**Comment by Kybeer** [comment]
Hi have the same issues on Windows 10 clients. The session disconnects after a few seconds and I can see 'mrwebrtc.DLL' crashing in Event Log/Applicayion Log. 
It doesn't matter if I disable WebRTC in the server settings or not.
